editor/inserter/menu.js

```diff
@@ -33,6 +33,7 @@ class InserterMenu extends Component {
 		this.onKeyDown = this.onKeyDown.bind( this );
 		this.getVisibleBlocks = this.getVisibleBlocks.bind( this );
 		this.sortBlocksByCategory = this.sortBlocksByCategory.bind( this );
+		this.debouncedSpeak = debounce( this.speakAssertive.bind( this ), 500 );
```

Comment:
Hm not sure there's really the need for `.cancel()`, what is the use case? Like search, and then immediately close the inserter before 500ms have passed? Will try it though.

I'd totally agree to make the delay for this kind of functions that need to wait for users when they stop typing a constant.